src/App.jsx

```diff
@@ -8,30 +8,37 @@ import Home from "./components/Home";
 import Login from "./components/Login";
 import Navigate from "./components/Navigate";
 import Register from "./components/Register";
-import SiteNav from "./components/SiteNav";
 import NotFound from "./components/NotFound";
+import NavBar from "./components/NavBar";
 
 function App() {
-  return (
-    <Fragment>
-      <SiteNav />
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  const handleLogin = () => {
+    setIsLoggedIn(true);
+  };
+
+  const handleLogout = () => {
+    setIsLoggedIn(false);
+  };
 
-      <BrowserRouter>
+  return (
+    <BrowserRouter>
```

Comment:
What's the purpose of moving the NavBar and Footer inside of BrowserRouter?

Reply:
The NavBar has to be within the BrowserRouter component based on my research. If it's not inside the BrowserRouter component, it won't be able to properly handle the navigation links.